
try:
    x = 2;
    raise Exception;
except:
    y = 22;


print(y);